docker/build.sh

```diff
@@ -2,22 +2,44 @@
 
 set -e
 
-if [[ $# -ne 1 ]]; then
-  echo "Expected 1 parameter, but got $#."
+# Make all paths relative to the dir of this script.
+cd "$(dirname "$0")" > /dev/null
+
+if [[ $# -ne 1 ]] && [[ $# -ne 2 ]]; then
+  echo "Expected 1 or 2 parameters, but got $#."
 
   cat <<USAGE_DOC
-Usage: build.sh <VERSION>
+Usage: build.sh <VERSION> [<TARGET>]
+
+Builds the Gitkv docker image for the given version and tags the container as
+'intenthq/gitkv:<VERSION>'.
 
-Builds the gitkv binary for the given version and tags the container as 'gitkv:<VERSION>'.
+The binary is copied from the Cargo release build for the given target.
 
 Parameters:
-    VERSION: The version to build, in the format 'vX.Y.Z' — note the 'v' prefix is needed.
+    VERSION: The version to build, in the format 'vX.Y.Z' — note the 'v' prefix
+             is needed.
+    TARGET:  The binary target triple to build into the image,
+             eg. 'x86_64-unknown-linux-musl'. If not given, will assume this
+             machine's target triple.
 USAGE_DOC
   exit 1
 fi
 
 VERSION=$1
 
-echo "Building gitkv:$VERSION"
+if [ -z $2 ]; then
```

Comment:
This is just to allow it to work locally (where you don't use cross-compilation) and remotely on Travis (where you do).